### PR TITLE
Issue #540 resolved

### PR DESCRIPTION
**Pull request by supr158**
Issue resolved
->Added a 'required' field in html input field for worker report file.
->Exception handling in case unsupported file format is uploaded.
Screenshot from 2022-01-27 21-32-28
->Fixed the notice board UI
Screenshot from 2022-01-27 21-30-24

### CODE REVIEW

**Comment by MananJethwani** [comment]
@supr158 kindly ask your review team member to review this PR asap

**Comment by supr158** [comment]
@anurag0singh please review this

File: FusionIIIT/templates/hostelmanagement/notice_board.html [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {% load custom_tags %}
 {% block {{hall.hall_id}} %}
-    
+
     {% for notice in hall_notices|get_hall_no:hall_id %}
-        <div class="ui icon warning message">
-        <div class="ui raised segment">
+
+        <div style="margin-top: 20px;border-style: outset;border-radius: 7px;border-width: 2.5px;"
```
Comment (anurag0singh) [inline FusionIIIT/templates/hostelmanagement/notice_board.html]: I don't think that inline CSS is a good idea here. I would like you to find the common CSS file and see if you can solve this issue from there. If not possible then we can discuss this further.

**Review by anurag0singh — COMMENTED** [review]
Please address this idea if possible to do so in the common CSS.

File: FusionIIIT/templates/hostelmanagement/notice_board.html (hunk 1 above)
Comment (supr158) [reply FusionIIIT/templates/hostelmanagement/notice_board.html]: I have removed the inline css.You can check it.

**Review by anurag0singh — APPROVED** [review]
Really nice work guys. PR approved.

**Comment by anurag0singh** [comment]
@MananJethwani Please merge this PR.

**Pull request merged**